### PR TITLE
Improve: .ocamlformat-enable files listing files to format when ocamlformat is disabled by default

### PR DESCRIPTION
**Pull request by gpetiot**
This is an alternative to #851

### CODE REVIEW

File: src/Conf.ml [hunk 1]
```diff
@@ -488,7 +488,14 @@ let info =
...
          $(b,.ocamlformat-ignore) file. Lines starting with $(b,#) are \
-         ignored and can be used as comments." ]
+         ignored and can be used as comments."
+    ; `P
+        "An $(b,.ocamlformat-enable) file specifies files that OCamlFormat \
+         should format even when the $(b,disable) option is set. Each line \
+         in an $(b,.ocamlformat-enable) file specifies a filename relative \
+         to the directory containing the $(b,.ocamlformat-enable) file. \
+         Lines starting with $(b,#) are ignored and can be used as \
+         comments." ]
```
Comment (avsm) [inline src/Conf.ml]: Which has precedence if a file is specified in both `.ocamlformat-enable` and `.ocamlformat-ignore`?

Comment (gpetiot) [reply src/Conf.ml]: With this implementation, `.ocamlformat-enable` is only checked when ocamlformat is disabled (`disable=true`), and `.ocamlformat-ignore` is only checked when ocamlformat is enabled (`disable=false`).

**Comment by rudihorn** [comment]
Thanks for writing this PR! I'm generally happy. Maybe an explicit precedence may be less confusing (i.e. defining an answer to @avsm's question), but I'm happy either way.

File: src/Conf.ml (hunk 1 above)
Comment (avsm) [reply src/Conf.ml]: That sounds reasonable -- would be good to define that in the manual pages.

Comment (gpetiot) [reply src/Conf.ml]: done

**Pull request merged**